internal/package_/workflow.go

```diff
@@ -60,6 +58,10 @@ type ProcessingWorkflowRequest struct {
 
 	// Location identifier for storing auto approved AIPs.
 	DefaultPermanentLocationID *uuid.UUID
+
+	// The global task queue used for starting the workflow
```

Comment:
Missing period at the end of the sentence. Maybe something like the following?

```suggestion
	// Task queues used for starting new workflows.
```